cli/main.rs

```diff
@@ -470,15 +470,21 @@ fn resolve_flags_and_init(
   }
 
   let default_v8_flags = match flags.subcommand {
-    // Using same default as VSCode:
-    // https://github.com/microsoft/vscode/blob/48d4ba271686e8072fc6674137415bc80d936bc7/extensions/typescript-language-features/src/configuration/configuration.ts#L213-L214
-    DenoSubcommand::Lsp => vec!["--max-old-space-size=3072".to_string()],
+    DenoSubcommand::Lsp => vec![
+      "--stack-size=1024".to_string(),
```

Comment:
Given that we need to always enable it, we should probably use it by default in `deno_core`. WDYT?

Reply:
its just a hack for now, i'm asking around about raising the default stack size in v8 to account for the recent change that caused this.